index.js

```diff
@@ -23,6 +30,9 @@ var allowedPolicies = [
  */
 function buildCSPString(policies, reportUri){
 	var cspString = Object.keys(policies).map(function(policyName){
+		if(policies[policyName] === true || policies[policyName].length === 0){
```

Comment:
I think if `policies[policyName]` isn't a boolean, string, or an array then there _should_ be error. Right? If it's only boolean then an empty string or an empty array will pass through resulting in an extra space in the CSP header. Consider the following settings:
```
{
  policies: {
    'block-all-mixed-content': true,
    'script-src': [],
    'style-src': [
      csp.SELF
    ],
  }
}
```
... will result in this with the proposed patch:
```
block-all-mixed-content; script-src; style-src 'self';
```
.. but if we remove empty string/empty array length check it will be this (notice the extra space):
```
block-all-mixed-content; script-src ; style-src 'self';
```